sagemaker_model_monitor/data_quality/nlp/data_drift_detection_mmd.ipynb

```diff
@@ -0,0 +1,1059 @@
+{
```

Comment:
<p>Typo: <code>excution</code> -&gt; <code>execution</code></p><p> </p><p>Grammar: <code>if there any data points that violates</code> - &gt; <code>if any data points violate</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Reply:
fixed